### ISSUE TITLE
cider-doc for java methods with array args rendered incorrectly

### ISSUE THREAD

**Issue by bsless** [clojure-emacs/orchard#278]
## Expected behavior

Call `cider-doc` on `java.nio.file.Path/of`, render docs such that `more` is represented as an array:
```
java.nio.file.Path/of
^Path [^String first, ^String<> more] ; or  [Ljava.lang.String;
^Path [^java.net.URI uri]
```

## Actual behavior

```
java.nio.file.Path/of
^Path [^String first, ^String more]
^Path [^java.net.URI uri]
```

## Steps to reproduce the problem

`M-x cider-doc` `java.nio.file.Path/of`

## Environment & Version information

```
Cider version 1.15.0
emacs --version
GNU Emacs 30.0.50
Development version 319e16550d66 on master branch; build date 2024-05-31.
```

### Clojure version

E.g. 1.11.1

### Java version

21

### Operating system

Pop OS 22.04

**Comment by vemv** [comment]
Thanks!

Some notes https://clojurians.slack.com/archives/C0617A8PQ/p1718605593241419

**Comment by alexander-yakushev** [comment]
As far as I can tell, this is already fixed in orchard 0.28.0/cider-nrepl 0.50.3 with #295.

<img width="368" alt="image" src="https://github.com/user-attachments/assets/9b5e96e7-9228-4e91-9e4d-18c2bd7fed6d" />

Thank you for the report, @bsless!
